cloud_governance/tag_user/tag_iam_user.py

```diff
@@ -38,7 +38,8 @@ def __write_into_csv_file(self, tag_keys: list, tag_values: dict):
         """
         with open(self.file_name, 'w') as file:
             file.write('Username, ')
-            tag_keys.remove('Username')
+            if 'Username' in tag_keys:
```

Comment:
Why you add it ?

Reply:
suppose some username has already tag Username. So I will delete it to add. previously all usernames are tagged with tagUsername

Reply:
So you are removing the user tag always ?

Reply:
That's why I am checking the Username. If it is there I will remove